src/sphinx/general/debugging.rst

```diff
@@ -0,0 +1,61 @@
+.. _debugging:
+
+#########
+Debugging
+#########
+
+Print Session Values
+====================
+
+If we wanted to print all values within a session we can just use this syntax:
+
+.. code-block:: scala
+
+    .exec { session => println(session); session }
+To print a single session value use syntax akin to this:
+  
+.. code-block:: scala
+
+    .exec({session => println(session("addComputer").as[String]); session }) 
+
+Logback
+=======
+
+There should be a default logback.xml file in the Gatling conf directory. 
+Add a file appender, similar to this:
```

Comment:
I added a file appender, so users could remove the none HTML response and view the page in a browser.